app/views/communication_logs/_form.html.erb

```diff
@@ -14,12 +14,13 @@
       <%= f.association :delivery_method, required: true, collection: ContactMethod.all %>
       <%= f.input :delivery_status, as: :select, required: true, collection: CommunicationLog::DELIVERY_STATUSES %>
       <%= f.input :needs_follow_up, as: :radio_buttons %>
+      <%= f.input :automated, as: :radio_buttons %> <!-- FIXME: block editing automated logs? -->
```

Comment:
Should we be able to edit auto-generated communication logs?

Reply:
I don't think so. Do you?

Reply:
Converted to an issue.